test/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM quay.io/podman/stable
+
+WORKDIR /project
+
+# Setup Yggdrasil & device-worker repo:
+RUN curl -k https://copr.fedorainfracloud.org/coprs/ondramachacek/yggdrasil/repo/fedora-34/ondramachacek-yggdrasil-fedora-34.repo -o /etc/yum.repos.d/yggd.repo
+RUN curl -k https://copr.fedorainfracloud.org/coprs/eloyocoto/k4e-test/repo/fedora-34/eloyocoto-k4e-test-fedora-34.repo -o /etc/yum.repos.d/k4e-agent.repo && \
```

Comment:
We should use `flotta` instead of `k4e` in names that we use for created files. (for example `k4e-agent.repo` -> `flotta-agent.repo`).